src/lib.rs

```diff
@@ -70,19 +70,21 @@ extern "C" {
 pub fn height_to_flags(height: u32) -> u32 {
 
     let mut flag = VERIFY_NONE;
-    if height > 170059 {
+    if height >= 170061 {
+        // bip16 enforce after timestamp > 1333238400 however latest tx violating is
+        // 6a26d2ecb67f27d1fa5524763b49029d7106e91e3cc05743073461a719776192 in block 170060
```

Comment:
I'm curious about where `170059` and `170061` come from in the first place? Why not just  `173805`?

Reply:
`170061` is what I found works in practice to complete a chain validation (ie no tx between 170061 and 173805 are violating bip16). But I do agree `173805` is the right value to have here